src/components/TipInput.vue

```diff
@@ -13,14 +13,23 @@
           v-if="!tipUrlStats.tokenTotalAmount.length || +tipUrlStats.totalAmountAe !== 0"
           :amount="tipUrlStats.totalAmountAe"
         />
+      </template>
+    </Component>
+    <Dropdown
+      v-if="tipUrlStats && tipUrlStats.tokenTotalAmount.length"
```

Comment:
Shouldn't we hide this dropdown on UserProfile page?

Reply:
I guess we should be able to send fungible tokens to user profiles also. @mradkov What do you think?

Reply:
This is a good question, it is not included in the designs so I will hide it for now.

Reply:
I think it could go into the "stats" section.